services/scrape-test/app/routes.py

```diff
@@ -0,0 +1,79 @@
+# routes.py
+
+from fastapi import APIRouter, BackgroundTasks, HTTPException, Query, Path
+from models import JobCreateResponse, JobStatusResponse
+from scraper import process_job
+from logging_config import logger
+import global_vars
+import uuid
+
+router = APIRouter()
+
+
+@router.post(
+    "/start_job",
+    response_model=JobCreateResponse,
+    summary="Start a new scraping job",
+    tags=["Jobs"],
+)
+async def start_job(
+    advertiser_id: str = Query(
+        ..., description="Advertiser ID to process", example="AR00642912486307135489"
+    ),
+    background_tasks: BackgroundTasks = None,
+):
+    """
+    Start a new scraping job for a given advertiser ID.
+
+    Args:
+        advertiser_id (str): The ID of the advertiser to process.
+        background_tasks (BackgroundTasks): FastAPI background tasks.
+
+    Returns:
+        JobCreateResponse: Contains the job ID of the started job.
+    """
+    job_id = str(uuid.uuid4())
+    global_vars.job_statuses[job_id] = "Pending"
+    global_vars.active_jobs.add(job_id)
```

Comment:
not a fan of global variables. I'ld use some in memory storage like Redis or something similiar

Reply:
But your solution is okay, it's just matter of opinions :D